README.md

```diff
@@ -31,6 +31,12 @@ Step 5. ```vagrant up```
 
 From here, you should be able to browse to http://localhost:8080/. The website is served with NGINX and PHP 7.
 
+### Getting started with Docker
+If you're a Docker user rather than Vagrant, there is a `docker-compose set up provided`.
```

Comment:
Did you mean "`docker-compose` set up provided"? I'm not quite familiar with Docker.